setup.py

```diff
@@ -154,15 +154,16 @@ def build_dir():
 BUILD_DEPENDS = glob.glob(path_join("include", "khmer", "_cpy_*.hh"))
 BUILD_DEPENDS.extend(path_join("include", "oxli", bn + ".hh") for bn in [
     "khmer", "kmer_hash", "hashtable", "labelhash", "hashgraph",
-    "hllcounter", "khmer_exception", "read_aligner", "subset", "read_parsers",
-    "kmer_filters", "traversal", "assembler", "alphabets", "storage"])
+    "hllcounter", "oxli_exception", "read_aligner", "subset", "read_parsers",
+    "kmer_filters", "traversal", "assembler", "alphabets", "storage",
+    "oxli_exception_convert"])
```

Comment:
I added `oxli_exception_convert.hh` to `src/include/oxli`, but the implementation (in `src/khmer/oxli_exception_convert.cc`) depends on `Python.h`. Should we keep both header and implementation inside `khmer/_oxli` instead (since they are extension code, and not necessary for the C++ library)?